install.sh

```diff
@@ -770,10 +713,12 @@ install_buildah(){
       ;;
   esac
 
+  is_command_exists usermod || abort "usermod not found" 
+   
   # Configure user namespaces
   if ! grep -q "$(get_user)" /etc/subuid; then
```

Comment:
Also, let's avoid simple greps where possible. This code will not work as intended if there are entries for the user `user2` in `/etc/subuid`, while we are grepping for the user `user`.

Use awk or cut to pick the first field from file, then try exact match without any `*` or like.